Providers/Scripts/2.4x-2.5x/Scripts/nxOMSPlugin.py

```diff
@@ -277,13 +321,18 @@ def get_immediate_subdirectories(a_dir):
         return []
 
             
-def copy_all_files(src, dest):
+def copy_all_files(src, dest, is_exec):
     try:
         src_files = os.listdir(src)
         for file_name in src_files:
-            full_file_name = os.path.join(src, file_name)
-            if os.path.isfile(full_file_name):
-                shutil.copy(full_file_name, dest)
+            full_src_file = os.path.join(src, file_name)
+            full_dest_file = os.path.join(dest, file_name)
+            if os.path.isfile(full_src_file):
+                shutil.copy(full_src_file, dest)
+                if is_exec:
```

Comment:
This exec action should be taken out of this try/except block. If the copy succeeds but os.stat or os.chmod fails, we should log a different message.